tz.cpp

```diff
@@ -410,23 +410,25 @@ load_timezone_mappings_from_csv_file(const std::string& input_path)
     for (int i = 0; i < 4; ++i)
         getline(is, copyright);
     
```

Comment:
I updated the pull request. I also added a corresponding reload_tzdb for consistency.